docs/reference/template_tags.md

```diff
@@ -21,7 +21,7 @@ Import as
 
 
 
-<a href="https://github.com/EmilStenstrom/django-components/tree/master/.venv/lib/python3.11/site-packages/django_components/templatetags/component_tags.py#L117" target="_blank">See source code</a>
+<a href="https://github.com/EmilStenstrom/django-components/tree/master/src/django_components/templatetags/component_tags.py#L117" target="_blank">See source code</a>
```

Comment:
Did you have to fix this manually, or did this get updated when you ran `mkdocs build` or `mkdocs serve`?

Reply:
This got changed/fixed when running `hatch run docs:build`